### ISSUE TITLE
Run VRT daily for teamsite pages

### ISSUE THREAD

**Issue by rianfowler** [department-of-veterans-affairs/va.gov-team#158]
Migrated from: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16518

tl;dr 
- we'd like to run `npm run test:vrt` from `vets-website` repo daily and report success / failure

Details 
- README.md about what this script does can be found [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/proxy-rewrite/README.md#teamsite-visual-regression-testing). A short deck about it can be found [here](https://docs.google.com/presentation/d/1f_NY551TmTzG5Css9qXXlAN-bZ25SPJNG15HwFnRVAI/edit?usp=sharing).
- Several VA subdomain sites like www.benefits.va.gov that are not directly under our control use the va.gov header. They run a production build javascript file hosted in `vets-websites`'s S3 that removes the existing header and starts a slightly modified version of the va.gov header. 
- This header shares code with the va.gov website. Changes made to the va.gov header are reflected on all of the teamsite pages
- We're want to run a visual regression testing suite daily that will compare past screenshots of the header/footer against the current versions on the teamsite pages. 

Rationale
- This regression testing doesn't fit cleanly into the continuous integration flow
  - the tests are slow- taking 1-3 seconds each and half a minute for a small suite 
  - regressions can occur outside of the context of CI triggers like code changes (e.g. an update to Teamsites which hosts content outside of `vets-website` can cause a regression)

Potential Approach
- Running this on Jenkins seems like the right place
  - the script requires the same environment as `vets-website` builds e.g. it needs to run `yarn install` before running the script 
  - Initially we think the job could report the success / failure to the DSVA#platypus-tools-team channel in slack 

Long Term Considerations
- Failures generate a diff image that we'll want to capture and include in the report from the jenkins job. [Example](https://github.com/department-of-veterans-affairs/vets-website/raw/master/src/applications/proxy-rewrite/readme-assets/diff-image-example.png)
- We want a better long term reporting mechanism - e.g. a failure could create an issue in github. Open to suggestions / ideas on feasibility here. 
- We aren't sure how this test suite will scale. There's interest to allow other apps teams create their own VRT but there are some considerations that need to be made 
  - the `vets-website` repo will balloon in size if we store hundreds of the baseline images in it. We are considering hosting the images externally and pulling them in as part of this script (e.g. from S3 or a separate github repo)

**Comment by wyattwalter** [comment]
hey @rianfowler is this something that we still want to try to enable?

**Comment by rianfowler** [comment]
@wyattwalter Not anymore. I think circle supports scheduled jobs that would meet our infrastructure needs. I removed the operations label.
